python/paddle/fluid/layers/control_flow.py

```diff
@@ -2128,62 +2329,88 @@ def memory(self,
                need_reorder=False,
                dtype='float32'):
         """
-        Create a memory variable for dynamic rnn.
+        Create a memory Variable for DynamicRNN to deliver data cross time steps.
+        It can be initialized by a existing Tensor or to a constant Tensor of given
+        dtype and shape.
 
-        If the :code:`init` is not None, :code:`memory` will be initialized by
-        this variable. The :code:`need_reorder` is used to reorder the memory as
-        the input variable. It should be set to true when the initialized memory
-        depends on the input sample.
+        Args:
+            init (Variable, optional): LoDTensor used to initialize the memory.
+                If init is not None, it should hold the same number of sequences
+                as RNN's input (the input LoDTensor set by :code:`step_input()` )
+                and the memory will be initialized to it. If init's LoD is null,
+                it will be treated as a minibatch with :code:`init.shape[0]` sequences
+                of length 1. The default value is None.
+            shape (list|tuple, optional): When init is None, it is used to specify
+                the memory's shape. Note that the shape does not include the batch_size.
+                If setting shape to :code:`[D1, D2, ...]` , the shape of memory Tensor
+                will be :code:`[batch_size, D1, D2, ...]` , where batch_size is
+                determined by RNN's input sequences. The default value is None.
+            value (float, optional): When init is None, it is used as initalized value
+                of memory. The default value is 0.0.
+            need_reorder (bool, optional): When init is not None, it determines whether
+                the memory need to reorder like the RNN's input sequeneces. It should be
```

Comment:
"the memory need to" -> "the memory needs to"

Reply:
Done.